packages/angular_devkit/build_angular/src/angular-cli-files/plugins/postcss-cli-resources.ts

```diff
@@ -64,11 +72,42 @@ export default postcss.plugin('postcss-cli-resources', (options: PostcssCliResou
       return cachedUrl;
     }
 
+    if (inputUrl.startsWith('~')) {
```

Comment:
This block should go.  It was meant to bridge the gap for 6.x by providing rudimentary support.  It doesn't actual work properly either.  Instead just remove the tilde and continue processing.

Reply:
Updated.